docs/getting_started.md

```diff
@@ -0,0 +1,166 @@
+We have already incorporated the `miQC` results into the {ref}`colData() of the SingleCellExperiment object<sce_file_contents:cell metrics>`. 
+All cells that are identified as low-quality cells will have `FALSE` in the `miQC_pass` column of the `colData()`. 
+
+The following commands can be used to remove those cells prior to downstream analysis: 
+
+```r
+# create a vector of barcode names for cells that are not compromised 
+# the `$` notation denotes access to the colData slot of the `SingleCellExperiment` object 
+cells_to_keep <- colnames(sce$miQC_pass == TRUE)
+
+# filter the `SingleCellExperiment`
+filtered_sce <- sce[, cells_to_keep]
+```
+For more information on using `miQC` for filtering cells, see the following resources: 
+- [`miQC` vignette](https://bioconductor.org/packages/release/bioc/vignettes/miQC/inst/doc/miQC.html)
+- [Hippen _et al._ 2021](https://doi.org/10.1371/journal.pcbi.1009290)
+
+Alternatively, you can directly filter cells based on having a minimum number of unique genes, total reads, and a maximum fraction of mitochondrial reads. 
+Using the function, [`scuttle::addPerCellQCMetrics()`](https://rdrr.io/github/LTLA/scuttle/man/addPerCellQCMetrics.html), we can calculate the total RNA counts, unique genes, and mitochondrial fraction for each cell and store the results in the `colData()`. 
+These metrics can be used to directly filter the `SingleCellExperiment` object based on informed minimum thresholds. 
+The `filtered.rds` files contain `SingleCellExperiment` objects with these metrics added to the `colData()` and you can read more about all metrics that are included in the {ref}`cell metrics section of the SingleCellExperiment file contents<sce_file_contents:cell metrics>`.
+
+If you are planning to filter low quality cells using these hard cutoffs, we encourage you to read more about the various metrics and plot the distribution of each metric before deciding on which cells to exclude. 
+The chapter on [Quality Control in Orchestrating Single Cell Analysis](http://bioconductor.org/books/3.13/OSCA.basic/quality-control.html#quality-control) provides a nice guide to checking diagnostic plots and then choosing cutoffs. 
+
+## Normalization 
+
+Single-cell RNA sequencing inherently comes with technical confounders that, if not accounted for, can impact the interpretation of downstream analysis. 
+Technical variables include differences in sequencing depth and cell size across cells in a single-cell library ([Stegle _et al._ 2015](https://doi.org/10.1038/nrg3833).)
+
+If we were to proceed with downstream analysis such as differential gene expression and cell clustering without accounting for such effects, these technical problems could impact the overall conclusions.
+We need to normalize our data to account for the technical variation between cells in our data and maximize biological variation, allowing us to make more accurate conclusions from downstream analysis.
```

Comment:
I'm torn about whether we need this much information here or if it's better to get right into what we recommend. We do provide links out to other resources.

Reply:
> I tend to think this document should have less rather than more

I agree with this take and would extend it to maybe mean cutting the downstream analysis portion.

Reply:
To clarify, do you mean completely remove the downstream analysis portion, or to just simplify it?

Reply:
I meant completely remove, but after taking a look at @jashapiro's comments I propose: you start by responding to his comments (e.g., removing feature selection) and then we'll see if we want to cut further.